<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerioViewClass</class>
 <widget class="QWidget" name="PerioViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>1424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1243</width>
    <height>1424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PerioView</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>1195</width>
     <height>801</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="PerioStatusView" name="maxilla"/>
   <widget class="PerioStatusView" name="mandibula"/>
  </widget>
  <widget class="HalfRoundedButton" name="upperButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Горна зъбна редица</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="HalfRoundedButton" name="lowerButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Долна зъбна редица</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Костна загуба:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="boneSpin">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="systemicCheck">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Системни фактори</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="restoreCheck">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>140</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Нужда от цялостно възстановяване на захапката</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Пушене:</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioSmoke_1">
        <property name="text">
         <string>Непушач</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSmoke_2">
        <property name="text">
         <string>Бивш пушач</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioSmoke_3">
        <property name="text">
         <string>до 10 цигари на ден</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSmoke_4">
        <property name="text">
         <string>до 20 цигари на ден</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSmoke_5">
        <property name="text">
         <string>над 20 цигари на ден</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="riskLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>1300</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PerioStatisticView" name="perioStatistics" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>1050</y>
     <width>1121</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="DateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string notr="true">dd.MM.yyyy 'г.'</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата:</string>
   </property>
  </widget>
  <widget class="PatientTileInfo" name="patientInfoTile" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>811</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <zorder>patientInfoTile</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>upperButton</zorder>
  <zorder>lowerButton</zorder>
  <zorder>label</zorder>
  <zorder>boneSpin</zorder>
  <zorder>systemicCheck</zorder>
  <zorder>restoreCheck</zorder>
  <zorder>groupBox</zorder>
  <zorder>riskLabel</zorder>
  <zorder>perioStatistics</zorder>
  <zorder>dateEdit</zorder>
  <zorder>label_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>View/uiComponents/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>PatientTileInfo</class>
   <extends>QWidget</extends>
   <header>View/SubWidgets/PatientTileInfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PerioStatusView</class>
   <extends>QWidget</extends>
   <header>View/Widgets/PerioStatusView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HalfRoundedButton</class>
   <extends>QPushButton</extends>
   <header>View/uiComponents/HalfRoundedButton.h</header>
  </customwidget>
  <customwidget>
   <class>PerioStatisticView</class>
   <extends>QWidget</extends>
   <header>View/SubWidgets/PerioStatisticView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
